packages/graphback/src/resolvers/index.ts

```diff
@@ -29,10 +29,6 @@ export class ResolverGenerator {
       hasCustomElements = true
     }
 
-    return {
-      types: generateResolvers(this.context),
-      index: generateIndexFile(this.context, hasCustomElements),
-      custom: generateCustomResolvers(customContext)
-    }
+    return generateApolloResolvers(this.context, customContext, hasCustomElements)
```

Comment:
How about not call it apollo but create.interface and document that this format is aligned.with apollo